tox.ini

```diff
@@ -92,9 +92,8 @@ deps =
     certifi; platform_system == "win32" or platform_system == "Darwin"
     typing_extensions
 commands =
-    mypy --install-types --non-interactive --strict bson
```

Comment:
Why loose `--strict` testing on bson?

Reply:
We'll pick it back up in [PYTHON-3698](https://jira.mongodb.org/browse/PYTHON-3698), this simplifies our config.